python/saltedge.py

```diff
@@ -46,10 +46,10 @@ def sign(self, message):
     def generate_signature(self, method, expire, some_url, payload=""):
         """
         Generates base64 encoded SHA256 signature of the string given params, signed with the client's private key.
-        :param method:  uppercase method of the HTTP request. Example: GET, POST, PATCH, PUT, DELETE, etc.;
-        :param expire:  the full requested URL, with all its complementary parameters;
-        :param some_url: the request post body. Should be left empty if it is a GET request, or the body is empty;
-        :param payload: the uploaded file digested through MD5 algorithm. Should be left empty if it is a GET request, or no file uploaded.
+        :param method: uppercase method of the HTTP request. Example: GET, POST, PATCH, PUT, DELETE, etc.;
+        :param expire: timestamp of when signature will expire
```

Comment:
request expiration time as a UNIX timestamp in UTC timezone. We suggest to use +1 minute from the current time. The maximum value is 1 hour from now in UTC